addons/fortify/XEH_postInit.sqf

```diff
@@ -63,3 +64,18 @@ GVAR(objectRotationZ) = 0;
         [_object, 0, ["ACE_MainActions"], _removeAction] call ACEFUNC(interact_menu,addActionToObject);
     };
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(setMarkerVisible), {
+    params ["_object"];
+    [
+        {!isNull player},
+        {
+            params ["_object"];
+            if (GVAR(markObjectsOnMap) isEqualTo 1 || {[_object getVariable QGVAR(objectSide), side group player] call BIS_fnc_sideIsEnemy}) then {
+                private _marker = _object getVariable QGVAR(mapMarker);
+                _marker setMarkerAlpha 0;
+            };
+        }, 
+        _object
+    ] call CBA_fnc_waitUntilAndExecute;
```

Comment:
There must be a better way to do this.

Reply:
Gotta wait somehow for the player object to exist for JIP side checking.